README.md

```diff
@@ -98,33 +102,64 @@ For TDP Vagrant usage see https://github.com/TOSIT-IO/tdp-vagrant.
 ./scripts/helper.sh -h
 ```
 
-### Services Deployment
+### Configure prerequisites
 
-#### Main playbook
+To use `tdp-collection-prerequisites` it is necessary to use the `-e prerequisites` option when using `setup.sh`.
 
-```
-ansible-playbook deploy-all.yml
+```bash
+ansible-playbook ansible_roles/collections/ansible_collections/tosit/tdp_prerequisites/playbooks/all.yml
 ```
 
-This playbook deploys the following services: a CA, an LDAP, a KDC, PostgreSQL, ZooKeeper, Hadoop core (HDFS, YARN, MapReduce), Ranger, Hive, Spark (2 and 3), HBase and Knox.
+This playbook deploys the following services: Chrony, a CA.
 
-#### SSH Key Generation and Deployment
+#### HTTP proxy configuration
 
-It is **optionally** possible to generate a new ssh key pair and deploy the public key to each host, though `vagrant ssh <ansible-host>` works just fine in the context of this getting-started cluster. Use the below command to generate SSH keys and deploy them throughout the cluster:
+You can configure HTTP proxy with Ansible variables:
+- `http_proxy`
+- `https_proxy`
+- `no_proxy`
```

Comment:
It's a bit unclear what these variables will do, this part needs more context IMO

Reply:
I just noticed it is documented at `inventory/group_vars/all.yml`, we don't need this in the main README IMO

Reply:
@leopaul36 I remove the "HTTP proxy configuration" section ?

Reply:
This means, people using this, can't get information about proxy from the readme ?

I think, at least, there should be a mention to proxy in the readme. (even if it's just telling you to go to the group_vars)

Reply:
Documentation added with https://github.com/TOSIT-IO/tdp-collection-prerequisites/pull/9.